### PR TITLE
Add blurb about obligation flag.

### PR DESCRIPTION
**Pull request by JayMurdoch**
Add blurb about obligation flag similar to must support blurb, based on Agency QA feedback

### CODE REVIEW

**Review by dt-r — CHANGES_REQUESTED** [review]
Rethink what this looks and reads like to determine the method of injection of that obligation sentence. See further information in the inline comment.

File: input/pagecontent/general-requirements.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ Labelling an element *[Must Support](https://www.hl7.org/fhir/conformance-rules.
...
 
-In AU Core, the meaning of *Must Support* is specified in terms of [Obligation Codes](https://hl7.org/fhir/extensions/CodeSystem-obligation.html) in [obligation extensions](https://hl7.org/fhir/extensions/StructureDefinition-obligation.html) on the element definition. The obligation codes used to define the minimum obligations of *Must Support* elements in this implementation guide are reiterated below.
+In AU Core, the meaning of *Must Support* is specified in terms of [Obligation Codes](https://hl7.org/fhir/extensions/CodeSystem-obligation.html) in [obligation extensions](https://hl7.org/fhir/extensions/StructureDefinition-obligation.html) on the element definition. When rendered in an implementation guide each profile is presented in different formal views under tabs labelled “Differential Table”, “Key Elements Table”, and “Snapshot Table”. The elements with obligations in these views are flagged with an <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" title="This element has obligations">O</span>.
```
[the remark this answers is not in the document]
Comment (JayMurdoch) [reply input/pagecontent/general-requirements.md:66]: Shifted to same place as Must Support element. 

Is there a need to add detail of how obligations are rendered in raw also?

[the remark this answers is not in the document]
Comment (dt-r) [reply input/pagecontent/general-requirements.md]: Re your question above - depends on what you were trying to add. I'm reviewing whether you a) achieved what the blurb of this PR is (which is what you say it is) and b) if after the change is introduced the IG is of equivalent or better quality.

File: input/pagecontent/general-requirements.md [hunk 2]
```diff
@@ -89,13 +89,13 @@ How the system processes the resource depends on local requirements that could a
 
 When a *Must Support* element requires a more tightly stated obligation, this obligation is specified in the AU Core Requester [obligation extension](https://hl7.org/fhir/extensions/StructureDefinition-obligation.html) on the element definition.
 
-#### Presentation of elements labelled Must Support in profiles
+#### Presentation of Must Support and obligations in profiles
 
-##### Presentation of elements labelled Must Support in table views
+##### Presentation of Must Support and obligations in table views
```
Comment (dt-r) [inline input/pagecontent/general-requirements.md:94]: Capitalise Obligation. Headings should be 
- Presentation of elements labelled Must Support and Obligation in profiles
- Presentation of elements labelled Must Support and Obligation in table views

File: input/pagecontent/general-requirements.md [hunk 3]
```diff
@@ -89,13 +89,13 @@ How the system processes the resource depends on local requirements that could a
...
 
 When rendered in an implementation guide each profile is presented in different formal views under tabs labelled "Differential Table", "Key Elements Table", and "Snapshot Table".
 
-The elements labelled *Must Support* in these views are flagged with an <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" title="This element must be supported">S</span>. Implementers should refer to the "Key Elements Table" to see the full set of elements that are mandatory or *Must Support*, and the full set of terminology requirements.
+The elements labelled *Must Support* in these views are flagged with an <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" title="This element must be supported">S</span>.  The elements with defined obligations in these views are flagged with an <span style="padding-left: 3px; padding-right: 3px; color: white; background-color: red" title="This element has obligations">O</span>. Implementers should refer to the "Key Elements Table" to see the full set of elements that are mandatory or *Must Support*, and the full set of terminology requirements.
```
Comment (dt-r) [inline input/pagecontent/general-requirements.md:98]: Replace "The elements with defined obligations" to be "The elements labelled with Obligation in these views are"

Then go on to rephrase the rest of this section now that it is about Must Support and Obligation e.g. "Implementers should refer to the "Key Elements Table" to see the full set of elements that are mandatory or *Must Support* or with Obligations, and the full set of terminology requirements."

File: input/pagecontent/general-requirements.md [hunk 4]
```diff
@@ -89,13 +89,13 @@ How the system processes the resource depends on local requirements that could a
...
 
 Implementers should take note that the full set of constraints (i.e. invariants) are only presented in the "Detailed Descriptions" tab or the raw representation (e.g. XML or JSON) of the profile.
```
Comment (dt-r) [inline input/pagecontent/general-requirements.md:100]: This sentence on Implementers should take note is no longer true. It can be removed entirely.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments.

**Pull request closed without merging**